flake.nix

```diff
@@ -3,21 +3,7 @@
 
   inputs = {
 
-    ###########################################################################
-    # We pin the versions of haskell.nix and hackage.nix because
-    # - Cross compilation for windows is broken in newer versions.
-    # - plutus-preprocessor is a non-buildable package and it seems to be treated differently in newer versions.
-    # We should unpin this once we no longer need it by specifying only haskell.nix and let it use its own default hackage.nix
-    # haskellNix.url = "github:input-output-hk/haskell.nix";
-    hackageNix = {
-      url = "github:input-output-hk/hackage.nix?ref=for-stackage";
-      flake = false;
-    };
-    haskellNix = {
-      url = "github:input-output-hk/haskell.nix/a0283c855a38ed70ba521f7a9290e78488ddf11b";
-      inputs.hackage.follows = "hackageNix";
-    };
-    ###########################################################################
+    haskellNix.url = "github:input-output-hk/haskell.nix";
```

Comment:
Are you sure you want to make this change? It's the same one Aniket was trying to make in #5029 and it breaks the nix build due to a bug in newer versions of `haskell.nix`. IIRC, even `nix develop` is broken.

The problem with non-buildable packages has been fixed now but not the problem with licenses for Hackage packages. Although I mentioned both problems in my message to `#dx-haskell-nix` only one was addressed.

Reply:
I checked, and `nix develop` is OK. It's the nix build of `cardano-ledger-test:lib:cardano-ledger-test` that's broken, because it uses the `terminfo` package indirectly. This isn't a problem for us, if we don't mind ignoring Hydra failures until the problem is fixed in `haskell.nix`.